### PR TITLE
delete app-csa-privileged NS

### PR DESCRIPTION
**Pull request by umezawatakeshi**
see internal request NECOREQ-45

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: test/setup_test.go [hunk 1]
```diff
@@ -438,6 +438,16 @@ func applyAndWaitForApplications(commitID string) {
 		}
 	}
 
+	// TODO: remove this block after release the PR bellow
+	// https://github.com/cybozu-go/neco-apps/pull/2414
+	if doUpgrade {
+		_, _, err := ExecAt(boot0, "kubectl", "get", "ns", "app-csa-privileged")
+		if err == nil {
+			stdout, stderr, err := ExecAt(boot0, "kubectl", "annotate", "ns", "dev-kintone", "admission.cybozu.com/i-am-sure-to-delete=app-csa-privileged")
```
Comment (morimoto-cybozu) [inline test/setup_test.go:446]:
```suggestion
			stdout, stderr, err := ExecAt(boot0, "kubectl", "annotate", "ns", "app-csa-privileged", "admission.cybozu.com/i-am-sure-to-delete=app-csa-privileged")
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: test/setup_test.go (hunk 1 above)
Comment (umezawatakeshi) [reply test/setup_test.go:446]: oops...

**Review by morimoto-cybozu — DISMISSED** [review]
LGTM
Great utility!
We'll need `var _ = annotateNamespaceToDeleteDuringUpgrade` when deleting the caller.

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**